### DISCUSSION TITLE
Windows Keybind Guide

### DISCUSSION THREAD

**Discussion by AlanAmal** [amitmerchant1990/notepad#35] [Polls]
The Keybindings Guide Currently only shows the guide for Mac, should we add the Windows key binds to the guide. THe windows key binds work they just aren't spelled out on the guide.

**Comment by amitmerchant1990** [comment]
It's mentioned in the guide. Take a look.
![image](https://github.com/amitmerchant1990/notepad/assets/3647841/d1769b23-95fe-4efa-9ca6-6d4135e8eeee)
